vendor/github.com/aws/aws-sdk-go/service/rds/api.go

```diff
@@ -21378,6 +21378,7 @@ type DescribeDBClusterSnapshotsInput struct {
 	// when SnapshotType is set to shared. The IncludeShared parameter doesn't apply
 	// when SnapshotType is set to public.
 	SnapshotType *string `type:"string"`
+	Tags []*Tag `locationNameList:"Tag" type:"list"`
```

Comment:
Hmm. My guess is that if modifying the AWS Go SDK like this is working, this is likely a feature to be added in an upcoming AWS Go SDK update when the RDS service team officially publishes pending API model changes after a production rollout completes. Unfortunately though, please note that we cannot accept changes that are not officially published in the AWS Go SDK for support reasons and that changes like this to the `vendor` files will get automatically reverted during any future `go mod` operations.

For now please revert this pull request to using the available [`ListTagsForResource`](https://docs.aws.amazon.com/sdk-for-go/api/service/rds/#RDS.ListTagsForResource) call and we can get this functionality in. Thanks.

Reply:
I sent new commit about this problem but i'm not sure. maybe i misunderstand you. If i did so, i will try again fix this and thank you for your time.:)

Reply:
Hi @aybuke 👋 To clarify: we cannot accept changes to just the `vendor` directory contents. Any changes in the `vendor` directory must occur due to running `go mod vendor`.

This pull request needs to be changed to work without this new line here.